doc/releases/release-notes-3.2.rst

```diff
@@ -531,6 +532,7 @@ Drivers and Sensors
   * Enable the touch panel on the NXP MIMXRT1170 EVK.
 
 * LED
+  * Updated LEDPWM support for ESP32 boards
```

Comment:
This  needs an emptyline to render correctly.

Reply:
Oh KSCAN slipped in as well, could you sneak one there as well while you are at it? :-) Thanks!